docs/migration/manual-migration.mdx

```diff
@@ -311,14 +311,30 @@ following stylesheet:
 This is happening because `.another-selector` is unknown, so the solution is to
 import the stylesheet defining it:
 
-```scss title="theme/components/atoms/Example/Example.scss"
+````scss title="theme/components/atoms/Example/Example.scss"
 // add-next-line
 @import "theme/some/path/defining/another-selector";
 
 .example {
   @extend .another-selector;
 }
-```
+## Link component
+
+In V3, we now use Remix's own `Link` component from `@remix-run/react`. Although
```

Comment:
**issue:** we do, but developers shouldn't. They must keep using our atom. Can you detail the reasoning behind it? Any examples?

Reply:
Indeed, I actually wrote a doc on something absolutely unrelated to the real hightlighted issue in the comment... My brain was elsewhere it seems, I'll fix it.

Reply:
After investigation the related change mentioned in the description was not needed at all, and so does this MR. I'll close it.